app/views/provider/admin/messages/outbox/index.html.slim

```diff
@@ -21,7 +21,7 @@ h2 Sent Messages
       - @messages.each do |message|
         tr id="message_#{message.id}"
           td.select= bulk_select_one message
-          td = link_to message.subject, provider_admin_messages_outbox_path(message)
+          td = link_to message_subject(message), provider_admin_messages_outbox_path(message)
```

Comment:
I have added this to show the `(No subject)` message with the link, rather than not showing nothing. This will be only applied for already existing messages without subject/body.

See the discussion starting from https://issues.redhat.com/browse/THREESCALE-8393?focusedCommentId=20974783&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20974783